spec/opensearch_helper.rb

```diff
@@ -1,7 +1,7 @@
 module OpenSearchHelper
   def self.get_host_port
     if ENV["INTEGRATION"] == "true" || ENV["SECURE_INTEGRATION"] == "true"
-      "opensearch:9200"
+      "integration:9200"
```

Comment:
why is this changed?

Reply:
In the docker-compose file in build scripts, we have defined service name as "integration". So, the service url also needs to be updated. I will be pushing the build scripts in my next PR